lib/2wp-utils.js

```diff
@@ -16,6 +16,7 @@ const BTC_TO_RSK_MINIMUM_CONFIRMATIONS = 3;
 const MINIMUM_PEGIN_VALUE_IN_BTC = 1;
 const TO_BRIDGE_GAS_PRICE = 2;
 const BRIDGE_ADDRESS = '0x0000000000000000000000000000000001000006';
+const MINIMUM_PEGOUT_VALUE_IN_RBTC = 1;
```

Comment:
Just a suggestion: can you move it to line 17, so it gets below the`MINIMUM_PEGIN_VALUE_IN_BTC`?

Reply:
will do with next PR

Reply:
I don't think it's a good idea to put this here. That is not the minimum peg-out value in regtest.

The right path would be to have 2 values (legacy minimum peg-out value and minimum peg-out value) and a function that returns the corresponding one according to iris activation. The same way it is handled in the Bridge. Out of scope for this ticket but maybe @jeremy-then you could create a ticket for that. For the minimum peg-in value, we should get it directly from the Bridge.

For this task, I'd remove that value from here and just use it in the specific test you are working on